packages/office-ui-fabric-react/src/components/Button/SplitButton/SplitButton.styles.ts

```diff
@@ -183,6 +201,7 @@ export const getStyles = memoizeFunction(
             borderColor: 'GrayText',
             backgroundColor: 'Window',
           },
+          ...getEdgeChromiumNoHighContrastAdjustSelector(),
```

Comment:
pls check if this is needed too

Reply:
This is needed to fix the styling for the disabled splitbutton example. It is also applied to the span divider element in the disabled commandbar example, so I will need to add a style override in commandbar.styles that removes this to achieve the expected behavior